src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,54 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int[] arrayFrom = getIntArray(dateFrom.split("\\D"));
+        int[] arrayTo = getIntArray(dateTo.split("\\D"));
```

Comment:
It is better to use [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html) here